docs/userGuide/contentAuthoring.md

```diff
@@ -119,547 +110,10 @@ In addition, it supports:
-          </ul>
-        </div>
-      </li>
-    </ul>
-  </li>
-</ul>
-
-The site navigation bar has [responsive properties](https://www.w3schools.com/html/html_responsive.asp), and will collapse to a menu button when the screen width is smaller than 992 pixels. It will then be completely hidden when the screen size is smaller than 576 pixels.
-
-You are able to use [Markdown syntax](#supported-markdown-syntax), [Glyphicons](#glyphicons) and [Font Awesome icons](#font-awesome-icons) to author your site navigation layout.
-
-Note:
-- Only one navigation file can be specified in a page, and you must include its file extension.
-- There is no limit to the number of nested Dropdown menus, but each menu's height is restricted to 1000 pixels.
-    - Dropdown menu content exceeding 1000 pixels in height will be cut off.
-
-### Using Footers
-
-It is optional to put footers in your site. If you wish to do so, there are two ways that MarkBind supports.
-  
-1. [Specifying footer file in Front Matter](#specifying-footer-file-in-front-matter)
-2. [Defining your own inline footer](#defining-your-own-inline-footer)
-
-#### Specifying footer file in Front Matter
-
-To create a new footer, make a new markdown file (`.md`) in `_markbind/footers`.
-Author your footer contents and wrap everything in a `<footer>` tag.
-You can create more than one footer file if you want different footers for different pages.
-
-Tip: All new sites created with `init` will have a `footer.md` file automatically created for you as a template in `_markbind/footers`.
-
-```html
-<!-- In _markbind/footers/myFooterFile.md -->
-<footer>
-  This is my own footer!
-</footer>
-```
-
-After which, you must specify the file within a page's [front matter](#front-matter) `footer` attribute to render the footer.
-
-```html
-<!-- In the page that you want to include a footer -->
-<frontmatter>
-  footer: myFooterFile.md
-</frontmatter>
-```
-
-Note:
-- Only one footer file can be specified in the [front matter](#front-matter) per page, and you must include its file extension.
-
-#### Defining your own inline footer
-
-If you would like to have a page-specific footer content, you can author one inline with the same `<footer>` tag. 
-
-```html
-# Hello world
-<p>Some text</p>
-<footer>
-  <p align="center">
-    This is my inline footer, unique to this page!
-  </p>
-</footer>
-```
-
-Note:
-- Footer file specified in [front matter](#front-matter) overrides all inline footers in the same page.
-- Inline footers need to be the last element in a page.
-- If there is more than one inline footer in a page, only the last inline footer will be rendered.
-- Footers should not be nested in other components or HTML tags. If found inside, they will be shifted outside to be rendered properly.
-- [MarkBind components](#use-components) and [includes](#include-contents) are not supported in footers.
-
-### Heading Anchors

```

Comment:
This sub-section seems to have disappeared in the cleaned up version. :P

Reply:
This feature doesn't seem to be worthy enough to feature on the landing page.

I am OK with removing this section if there's no suitable place for it.